### ISSUE TITLE
Data at the root level is invalid. Line 1, position 1

### ISSUE THREAD

**Issue by Fossil01** [Sonarr/Sonarr#1312]
# Not sure if this is a Spotweb or Sonarr error. That's why i've posted this issue in both repositories.

I keep getting this error in the Sonarr logs when it tries to call Spotweb:

`Data at the root level is invalid. Line 1, position 1.`

It happens with every show I try to search for. This is the URL it calls:

`https://SPOTWEBURL/api?t=tvsearch&cat=5030,5040&extended=1&apikey=(removed)&offset=0&limit=100&rid=33759&tvmazeid=143&season=3&ep=5`

Here's the XML response when opening the URL in my browser:

```
<?xml version="1.0" encoding="utf-8"?>
<rss version="2.0" xmlns:atom="http://www.w3.org/2005/Atom" xmlns:newznab="http://www.newznab.com/DTD/2010/feeds/attributes/">
  <channel>
    <atom:link href="https://SPOTWEBURL/api" rel="self" type="application/rss+xml"/>
    <title>Spotweb Index</title>
    <description>Spotweb Index API Results</description>
    <link>https://SPOTWEBURL/</link>
    <language>en-gb</language>
    <webMaster>REMOVED</webMaster>
    <category/>
    <image>
      <url>https://SPOTWEBURL/images/spotnet.gif</url>
      <title>Spotweb Index</title>
      <link>https://SPOTWEBURL/</link>
      <description>SpotWeb Index API Results</description>
    </image>
    <newznab:response offset="0" total="0"/>
  </channel>
</rss>
```

**Comment by markus101** [comment]
Without seeing the full log of where the error is coming from it sounds like spot web returning something other than XML, but since the browser is getting XML back you'd need to intercept the response that Sonarr is getting back to see what its getting returned. Fiddler on Windows would be useful, but you could also use Wireshark.

Troubleshooting like this really belong on the forums though.

**Comment by Fossil01** [comment]
Roger. I'll look into it.

**Comment by Fossil01** [comment]
Okay I installed Sonarr on a Windows box and used Fiddler. No errors and I got results back from Spotweb and my other nZEDb based indexer without errors. The latter did not show in Sonarr on my Synology and the former threw the XML error before.

I click restart in Sonarr on my Syno, it crashed. I restarted it manually and the errors disappeared and no more Spotweb XML errors... Weird, but it seems fixed for now.
